src/pages/get-eth.tsx

```diff
@@ -182,309 +150,243 @@ const GetEthPage = ({
+                </InlineLink>
+              </p>
+              <p>{t("page-get-eth-dexs-desc-3")}</p>
+              <p>{t("page-get-eth-need-wallet")}</p>
+              <ButtonLink href="/wallets/find-wallet/">
+                {t("page-get-eth-get-wallet-btn")}
+              </ButtonLink>
+              <InfoBanner isWarning>
+                <Translation id="page-get-eth:page-get-eth-dexs-desc-4" />
+              </InfoBanner>
+            </Stack>
+
+            <Stack className="gap-4">
+              <h3 className="text-xl leading-6 md:text-2xl">
+                {t("page-get-eth-other-cryptos")}
+              </h3>
+              <p>{t("page-get-eth-swapping")}</p>
+              <CardList items={tokenSwaps} />
+              <InfoBanner isWarning>{t("page-get-eth-warning")}</InfoBanner>
+            </Stack>
+          </TwoColumnContent>
+        </Stack>
+
+        <Divider className="mx-auto my-16 md:my-32" />
+
+        <Stack className="gap-12">
+          <h2 className="text-2xl leading-6 md:text-3xl">
+            {t("page-get-eth-keep-it-safe")}
+          </h2>
+          <TwoColumnContent>
+            <Stack className="gap-4">
+              <Image
+                src={wallet}
+                className="mb-8 self-center"
+                sizes={walletImageWidth}
+                style={{ width: walletImageWidth, height: "auto" }}
+                alt=""
+              />
+              <h3 className="text-xl leading-6 md:text-2xl">
+                {t("page-get-eth-community-safety")}
+              </h3>
+              <CardList items={safetyArticles} />
+            </Stack>
+
+            <Stack className="gap-8">
+              <Stack className="gap-4">
+                <p>{t("page-get-eth-description")}</p>
+                <p>{t("page-get-eth-security")}</p>
+              </Stack>
+              <Stack className="gap-4">
+                <h3 className="text-xl leading-6 md:text-2xl">
+                  {t("page-get-eth-protect-eth-in-wallet")}
+                </h3>
+                <p>{t("page-get-eth-protect-eth-desc")}</p>
+                <InlineLink href="/wallets/">
+                  {t("page-get-eth-your-address-wallet-link")}
+                </InlineLink>
+              </Stack>
+              <Stack className="gap-4">
+                <h3 className="text-xl leading-6 md:text-2xl">
+                  {t("page-get-eth-your-address")}
+                </h3>
+                <p>{t("page-get-eth-your-address-desc")}</p>
+                <div className="mb-6 flex select-none flex-col-reverse justify-between rounded bg-[#191919] p-2 lg:flex-row">
+                  <p className="mb-0 font-monospace text-xs text-white">
+                    0x0125e2478d69eXaMpLe81766fef5c120d30fb53f
+                  </p>
+                  <p className="text-fail300 mx-4 mb-0 text-sm uppercase">
+                    {t("page-get-eth-do-not-copy")}
+                  </p>
```

Comment:
<img width="785" alt="image" src="https://github.com/user-attachments/assets/90457512-a8a1-4a38-b39d-5a50bc498b2c">

<img width="833" alt="image" src="https://github.com/user-attachments/assets/2e96c00d-4e09-4fe9-95cf-f02db6dc8e23">

I believe we want `text-error` here (`fail300` will not work in our current TW setup)

```suggestion
                  <p className="text-error mx-4 mb-0 text-sm uppercase">
                    {t("page-get-eth-do-not-copy")}
                  </p>
```